### ISSUE TITLE
Find a Store: No vocalization warning the link is external and will open in a new window

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13736]
**Describe the bug**
- There is no verbal indication to screen users that the "Get directions" button will open on a different website and different window, as opposed to the store title that opens more details for a given store on the same website.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Go to the Find a Store page
3. Click on the "View All Stores" button
4. Select any country
5. Tab to any location's "Get Directions" button
6. Hear screen reader feedback, missing the important detail that the link is external and will open in a new window.

**Expected behavior**
- On focusing the link, the screen reader vocalizes link purpose and the additional detail that it will open in a new window.
- Use `aria-label` to extend screen reader vocalization, as per screenshot.
- Suggested `aria-label`: _"Get Directions, this external link opens in a new window"_ please validate phrasing with @Manurod2018 and @vekterdude

**Screenshots**
Suggested markup:

![Screen Shot 2021-09-15 at 1 45 00 PM](https://user-images.githubusercontent.com/15113219/133488256-222b0cc7-befc-4451-a6e5-48a55f1c4baa.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus


![GetDirections](https://user-images.githubusercontent.com/45409515/144314003-9650d541-3b15-4eb1-a6b1-8f0122595886.png)

More extensive testing (on other screenreaders and devices) will be done with RC.
